src/components/VTranslationStatusBanner.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <VNotificationBanner v-show="!shouldHideBanner" @close="dismissBanner">
+  <VNotificationBanner
+    v-show="needsTranslationBanner"
```

Comment:
The `v-show` logic here only uses the translation percentage, and doesn't check for the value in localStorage. This is why if you dismiss a banner, and refresh the page, you will see the banner again, and will not be able to dismiss it anymore.

Reply:
That logic is handled in the `VNotificationBanner` component itself, with the local storage. But it seems like stacking `v-show` or something is messing with it. Thanks for catching this, I will debug it.

Reply:
So I think the issue is that because we have a `v-show` on the external render of the component, and because `v-show` just uses CSS to show/hide the element, it's overriding the local storage handling inside the component because `v-show="true"` in the component rendering it, even though inside (I've confirmed) `v-show="false"`.

The only thing I can think of is to forward the "v-show" variable as a new prop to be combined with the local storage variable. Hacky, but I guess it is necessary for `v-show` because of the way it works.

Reply:
There are two layers of `v-show` now, in the `TranslationBanner` and in the `NotificationBanner` (one checks for local storage, the other - for translation percentage). Maybe they are cancelling each other out? It would be nice to combine them, use the outer `v-show` value as a prop to `NotificationBanner`, maybe?

Reply:
The comment was too late, you've already implemented the fixes 🤣